src/angular/menu/menu.scss

```diff
@@ -32,7 +32,7 @@ sbb-menu {
     pointer-events: none;
```

Comment:
is this not too risky to change in this PR?

Reply:
(I mean all :where statements)

Reply:
That was unintentional